src/components/StatsGrid/StatsGridConfig.tsx

```diff
@@ -174,6 +244,15 @@ export const columnDefs = [
   {
     field: "bias",
     headerName: "Weight",
+    headerTooltip: (() => {
+      return [
+        htmlBold("Weigth of the node in the network."),
+        "This value indicates what is the probability of the node being selected to serve the traffic - the higher the value, the higher the probability.",
+        "This value is recalculated on every node registration based on number of factors including TTFB, Speedtest upload rate result, CPU load and number of cores, cache hit ratio, error rate and health check failures.",
+        "For nodes joining the network it can take several hours or days to reach the optimal value based on the nodes performance and network traffic.",
```

Comment:
```suggestion
        "For nodes joining the network it can take several days to reach the optimal value based on the nodes performance and network traffic.",
```